features/363-glossary/glossary.md

```diff
@@ -0,0 +1,48 @@
+Glossary
+========
+
+In this document, we define a list of terms to be used in communication.
+
+Objective
+---------
+
+In various documents, we will use different terms to refer to concepts in Skygear:
+- Documentation
+- Discussions
+- Emails/Announcements
+
+It would be clearer to use widely agreed terms to refer to specific concepts.
+
+Glossory
+--------
+- **Skygear Cluster**: the Skygear services (Gears, Gateway, and Controller), hosted on Kubernetes.
+- **Skygear Controller**: the management interface of Skygear Cluster. *Should not be used in public documents*.
+- **Skygear Portal**: the web management interface of Skygear Cluster.
+- **Skygear App**: a collection of microservices on Skygear Cluster, can omit 'Skygear' prefix if context if clear.
+- **Skygear Gateway**: the proxy which incoming traffic to Gears and Microservices must go through.
+- **Gear**: officially-supported multi-tenant-aware services.
+    - **Auth Gear**
```

Comment:
only have to take care documentation lang. Marketing terms can differ from doc terms